### PR TITLE
Markdwon previewer

### PR DESCRIPTION
**Pull request by shogof**
🎉 **Pull Request Description** 🎉

🚀 **Overview**  
I’m excited to share my latest project: the **Markdown Previewer**! This React application allows users to write Markdown and see the rendered output in real-time. It’s a simple yet powerful tool designed to make Markdown editing and previewing seamless. ✨

### **Features**
- **Live Markdown Preview:** As users type Markdown text into the editor, the preview updates dynamically to reflect the Markdown syntax.
- **Custom Styling:** Enjoy a clean and modern interface with smooth animations and responsive design. The editor and preview areas are styled to enhance readability and usability.
- **Responsive Design:** The application adapts to various screen sizes, ensuring a great experience on both desktops and mobile devices.

### **Styling**
- **Header:** Features a vibrant gradient background and a bold, clear title for a welcoming introduction.
- **Editor & Preview:** Both areas are styled with subtle animations and transitions to make the interface feel interactive and engaging.
- **Responsive Layout:** Utilizes media queries to adjust the layout for smaller screens, ensuring the editor and preview remain accessible and user-friendly.

### **Technical Details**
- **React Integration:** Built with React and `react-markdown` to handle the Markdown rendering.
- **Custom CSS:** Includes custom styles for a polished and professional look.

### **Notes**
This project is a part of my ongoing journey to improve my React skills. I’m eager to get feedback on the functionality, code quality, and overall design. Your suggestions and critiques are highly appreciated!

Thank you for reviewing my pull request. 😊

### CODE REVIEW

**Comment by shogof** [comment]
@royajamal 
**Hi dear kindly review my project**😄

**Comment by royajamal** [comment]
@shogof 
Great work! ❤️ You've successfully met the project requirements with excellence. Your feature is fully implemented! Well done and congratulations on a job well done! 🌟🎉

Status: Approved ✅
there is nothing  to say.

Highlights 😊
💯 The implementation of a great design is superb.
✔️ You've established a professional repository with your project's name.
✔️ You've mastered Gitflow by making the development branch your default and diligently committingy our work on a feature branch. 🌟
✔️ Add this to ensuring commits are more descriptive and follow a logical sequence.
✔️ Setup the linters correctly (You added the correct linters config into your project according tot he language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Build Markdwon Previewer project task and committed using the command line
✔️ You've initiated the pull request by submitting a link for review from your feature branch to the- development branch.
✔️ You have followed and completed the Markdwon Previewer and all requirements.
✔️ You've enriched your PR with a detailed description highlighting the features and improvementsi ncorporated in this pull request. 📝
✔️ pass all task in freecodecame
✔️ Your code is clean
✔️ Your repo has a professional README.md file
✔️ Your design is responsive
✔️   your project is deploy 
C heers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread ifs omething is not 100% clear.

File: src/MarkdownPreviewer.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  margin: 0;
+  font-size: 3rem;
+  font-weight: 700;
+  color: #fff;
+  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.editor-preview-container {
+  display: flex;
+  flex: 1;
```
Comment (pesinasiller) [inline src/MarkdownPreviewer.css:43]: very good code! maybe the next step could be implementing BEM in the styling

**Comment by ahzamir** [comment]
@shogof 

- Same issue with this project, you need to change the accessibility from deployment settings on vercel to make it accessible for everyone with the link

**Comment by shogof** [comment]
@ahzamir 
Kindly check it again :)

**Review by ahzamir — APPROVED** [review]
### Hi @shogof 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Markdown Previewer project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by shogof** [comment]
@ahzamir 
Thank you so much for your time and approval 😄

**Pull request merged**